src/EnergyPlus/PhotovoltaicThermalCollectors.cc

```diff
@@ -1450,11 +1444,10 @@ namespace PhotovoltaicThermalCollectors {
                   110.4)); // Sutherland's formula https://www.grc.nasa.gov/www/k-12/airplane/viscosity.html Sutherland's constant = 198.72 R
                            // converted to K =>110.4. At 273.15, Viscosity is 1.71E-5 as per Incropera, et al 6th ed. Temp range approx 273K - 373K
             k_air = 0.000000000015207 * std::pow(t_film + 273.15, 3.0) - 0.000000048574 * std::pow(t_film + 273.15, 2.0) +
-                    0.00010184 * (t_film + 273.15) - 0.00039333;                        // Dumas, A., and Trancossi, M., SAE Technical Papers, 2009
-            prandtl_air = 0.680 + 0.000000469 * std::pow(t_film + 273.15 - 540.0, 2.0); // The Schock Absorber Handbook, 2nd Ed. John C. Dixon 2007
-            density_air = 101.3 / (0.287 * (t_film + 273.15));                          // Ideal gas law
-            diffusivity_air = k_air / (cp_amb * density_air);                           // definition
-            kin_viscosity_air = mu_air / density_air;                                   // definition
+                    0.00010184 * (t_film + 273.15) - 0.00039333; // Dumas, A., and Trancossi, M., SAE Technical Papers, 2009
+            density_air = 101.3 / (0.287 * (t_film + 273.15));   // Ideal gas law
+            diffusivity_air = k_air / (cp_amb * density_air);    // definition
+            kin_viscosity_air = mu_air / density_air;            // definition
```

Comment:
[src/EnergyPlus/PhotovoltaicThermalCollectors.cc:1530]:(style),[redundantAssignment],Variable 'prandtl_air' is reassigned a value before the old one has been used.